workflow/envs/bcbio_variation_recall.yaml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bcbio-variation-recall=0.2.6
+  - bcftools=1.11
```

Comment:
seems to differ from docker version

Reply:
just checked the version `bcftools 1.11`